actioncable/lib/action_cable/subscription_adapter/redis.rb

```diff
@@ -2,7 +2,7 @@
 
 require "thread"
 
-gem "redis", "~> 3.0"
+gem "redis", "~> 3", "< 5"
```

Comment:
This needs to be `">= 3", "< 5"`; `"~> 3"` and `"~> 3.0"` both mean the same thing.